tests/ott_spec.py

```diff
@@ -10,7 +10,10 @@ def requirements(self):
         return project_path(), "oteltest"
 
     def environment_variables(self):
-        return {"OTEL_SERVICE_NAME": "my-svc"}
+        return {
+            "OTEL_SERVICE_NAME": "my-svc",
+            "OTEL_METRICS_ENABLED": "true",
```

Comment:
We must not introduce an env var with `OTEL_` prefix. These are reserved for OpenTelemetry (upstream)
Do we even need such configuration?
This is what we have in Go https://docs.splunk.com/observability/en/gdi/get-data-in/application/go/configuration/advanced-go-otel-configuration.html#exporters-configuration:

> `OTEL_METRICS_EXPORTER` - The metrics exporter to use. The default value is `otlp`. Accepted values are `otlp` and `none`. Setting `none` deactivates metric exports.

Reply:
Thanks -- these env vars are carried over from 1.x. I'm looking into removing at least some of them.